### PR TITLE
Adds Syndicate Frying Basket, role restricted cook traitor item.

### PR DESCRIPTION
**Pull request by Takahiru**
While sitting on the toilet, a revelation appeared to me, what if I just add the ability to fry whole corpses? So I tried.

Adds a 10 TC syndicate frying basket traitor item, it allows the frying of any corpse whole, also strips the clothing off of dead corpses.

:cl:  
rscadd: Added new things  
imageadd: added some icons and images  
/:cl:

frying basket

### CODE REVIEW

**Review by DoubleTapper — APPROVED** [review]

**Comment by Jarod1200** [comment]
Can you still throw the fried corpse and knock people down?

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 1]
```diff
@@ -102,7 +123,10 @@ God bless America.
 		else if(!frying && user.transferItemToLoc(I, src))
 			to_chat(user, "<span class='notice'>You put [I] into [src].</span>")
 			frying = new/obj/item/reagent_containers/food/snacks/deepfryholder(src, I)
-			icon_state = "fryer_on"
+			if(!superfry)
+				icon_state = "fryer_on"
+			else
+				icon_state = "syndie_fryer_on"
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm:128]:
```
icon_state = "fryer_on"
if(superfry)
    icon_state = "syndie_fryer_on"
```

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 2]
```diff
@@ -131,7 +155,10 @@ God bless America.
 		if(frying.loc == src)
 			to_chat(user, "<span class='notice'>You eject [frying] from [src].</span>")
 			frying.fry(cook_time)
-			icon_state = "fryer_off"
+			if(!superfry)
+				icon_state = "fryer_off"
+			else
+				icon_state = "syndie_fryer_off"
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm:158]:
```
icon_state = "fryer_off"
if(superfry)
       icon_state = "syndie_fryer_off"
```

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 3]
```diff
@@ -141,11 +168,30 @@ God bless America.
 			frying_burnt = FALSE
 			fry_loop.stop()
 			return
-	else if(user.pulling && user.a_intent == "grab" && iscarbon(user.pulling) && reagents.total_volume)
+
+
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]: Space

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 4]
```diff
@@ -141,11 +168,30 @@ God bless America.
...
+	if(user.pulling && user.a_intent == INTENT_GRAB && isliving(user.pulling))
+		if(superfry)
+			var/mob/living/H = user.pulling
+			if(H.stat == DEAD)
+				to_chat(user, "<span class ='notice'>You dunk [H] into [src],</span>")
+				frying = new/obj/item/reagent_containers/food/snacks/deepfryholder(src, H)
+				if(!superfry)
+					icon_state = "fryer_on"
+				else
+					icon_state = "syndie_fryer_on"
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm:176]:
```
icon_state = "fryer_on"
if(superfry)
       icon_state = "syndie_fryer_on"
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 5]
```diff
@@ -168,17 +165,14 @@ God bless America.
 			frying_burnt = FALSE
 			fry_loop.stop()
 			return
-
-
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]: One space was fine!

**Review by JamieD1 — APPROVED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm (hunk 5 above)
Comment (Takahiru) [reply code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]: heck

**Comment by Takahiru** [comment]
@jarod1200 no it acts like a normal fried item

**Comment by ilikesaying** [comment]
time to deep fry the clown

**Comment by Tipy1802** [comment]
10 TC is a bit much for what's essentialy a meme item. It does not really help you archiev your goals since to get fried the bodies need to be dead already. Actually using it may as well be a shooting yourself on the foot since it's very obvious when you use it (the grilles of the fryer become red) and just the exictance of fully fried corpses is a dead giveaway that you are a traitor.

Maybe make it cost 5 or 7 TC and maybe give it the abillity to fry people alive

**Comment by Takahiru** [comment]
I was already thinking about changing the cost, it is in function just a slightly better gibber because you don't need to strip the corpse.

**Comment by Hopekz** [comment]
> 10 TC is a bit much for what's essentialy a meme item.

This is a pretty good item from what I can tell. 
It allows you to fry and eat bodies whole to dispose of them with less trace than sticking them in a grinder.

**Comment by Jarod1200** [comment]
How hard can you valid the chef for having a red fry basket?

**Comment by Hopekz** [comment]
> How hard can you valid the chef for having a red fry basket?

If it was meant to be "hidden" I'd imagine it would not be easily distinguishable just by looking at it.

**Review by Hopekz — APPROVED** [review]

**Pull request merged**